package.json

```diff
@@ -34,6 +34,7 @@
     "webpack-cli": "^3.3.12"
   },
   "dependencies": {
+    "axios": "^0.20.0",
     "@fortawesome/fontawesome-svg-core": "^1.2.31",
     "@fortawesome/free-regular-svg-icons": "^5.15.0",
     "@fortawesome/free-solid-svg-icons": "^5.15.0",
```

Comment:
Ah, so this is how you got your fonts going! Very cool. I'll need to look into the library you're using.